parser.js

```diff
@@ -1415,6 +1417,7 @@
       //      \ b
 
       if (match('\\')) {
+        var res, from = pos;
```

Comment:
not 100% sure if `from` is correct here?

Reply:
I think it is because it's the position of `\\`.